Code/Framework/AzFramework/AzFramework/Visibility/OctreeSystemComponent.cpp

```diff
@@ -185,6 +185,28 @@ namespace AzFramework
         }
     }
 
+    void OctreeNode::Enumerate(const AZ::Frustum& includeFrustum, const AZ::Frustum& excludeFrustum, const IVisibilityScene::EnumerateCallback& callback) const
+    {
+        if (AZ::ShapeIntersection::Overlaps(includeFrustum, m_bounds) && !AZ::ShapeIntersection::Contains(excludeFrustum, m_bounds))
```

Comment:
Yup, looks good..

Reply:
Yeah, that's an option but in the end I don't think you'd end up with less code, just different code. It bugs me that this mostly duplicates the generic enumerate, but it's also pretty easy to understand what it's doing (you don't need to go look up what a FrustumShell is)